release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -235,6 +235,17 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-dev-exp"]
 === Developer experience
 
+[id="ocp-4-8-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-8-auth-tp-aws-sts"]
+==== Running {product-title} using AWS Security Token Service (STS) for credentials is generally available
+
+You can now use the Cloud Credential Operator (CCO) utility (`ccoctl`) to configure the CCO to use the Amazon Web Services Security Token Service (AWS STS). When the CCO is configured to use STS, it assigns components IAM roles that provide short-term, limited-privilege security credentials.
```

Comment:
Would "components _**with**_ IAM roles" sound better?

Reply:
I'm worried that phrasing is unclear (does it assign IAM roles to components, or assign components that have IAM roles to ____?), but I also don't love the current construction. How does this parse to you?

> When the CCO is configured to use STS, it assigns IAM roles that provide short-term, limited-privilege security credentials to components.

Reply:
Yes, I like that...adding "to components" at the end is the part I was missing. Thanks!